package.json

```diff
@@ -15,17 +15,17 @@
     "@patternfly/react-icons": "3.15.3",
     "@patternfly/react-table": "2.27.11",
     "@patternfly/react-tokens": "2.8.4",
-    "@redhat-cloud-services/frontend-components": "1.0.5",
+    "@redhat-cloud-services/frontend-components": "1.0.8",
     "@redhat-cloud-services/frontend-components-inventory-insights": "1.0.2",
     "@redhat-cloud-services/frontend-components-notifications": "1.0.1",
     "@redhat-cloud-services/frontend-components-remediations": "1.0.0",
     "@redhat-cloud-services/frontend-components-translations": "0.0.6",
     "@redhat-cloud-services/frontend-components-utilities": "1.0.0",
     "babel-plugin-react-intl": "5.1.18",
     "classnames": "2.2.6",
-    "react": "16.12.0",
-    "react-content-loader": "5.0.1",
-    "react-dom": "16.12.0",
+    "react": "16.13.0",
```

Comment:
OH wow you should really upgrade at least to versions 16.8.x  that is when the hooka were introduced.